sycl/include/syclcompat/atomic.hpp

```diff
@@ -199,42 +106,12 @@ template <typename T,
               sycl::access::address_space::global_space,
```

Comment:
Should sycl::access::address_space::global_space be sycl::access::address_space::generic_space?

Reply:
Yes! thanks again